src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -38,27 +38,17 @@ Removing or adding an authorized network on your {{ site.data.products.dedicated
 
 ### Establish GCP VPC Peering or AWS PrivateLink
 
-VPC peering is only available for GCP clusters, and AWS PrivateLink is only available for AWS clusters. 
+GCP VPC Peering and AWS PrivateLink allow customers to establish SQL access to their clusters entirely through private cloud infrastructure, without exposure to the public internet, affording enhanced security and performance.
 
-**Prerequisite:**
+VPC peering is available only for GCP clusters, and AWS PrivateLink is available for AWS clusters.
 
-You must create the VPC Peering or AWS PrivateLink connectionk in your GCP or AWS console.
-
-Refer to:
-- [Network Authorization for CockroachDB Cloud clusters: VPC Peering](network-authorization.html#vpc-peering)
-- [Network Authorization for CockroachDB Cloud clusters: AWS PrivateLink](network-authorization.html#aws-privatelink)
+To configure VPC Peering or PrivateLink, you create the private connection in your cloud provider, then configure your cluster to allow connections from the connection. For more information, refer to [Network Authorization for {{ site.data.products.dedicated }} clusters: GCP VPC Peering](network-authorization.html#vpc-peering) and [Network Authorization for {{ site.data.products.dedicated }} clusters: AWS PrivateLink](network-authorization.html#aws-privatelink).
```

Comment:
is there a way to rephrase "connections from the connection"? maybe "connections from the private endpoint"

Reply:
Done